internal/codegen/golang/templates/template.tmpl

```diff
@@ -1,4 +1,9 @@
-{{define "dbFile"}}// Code generated by sqlc. DO NOT EDIT.
+{{define "dbFile"}}
+{{if .BuildTags}}
+//go:build {{.BuildTags}}
+// +build {{.BuildTags}}
```

Comment:
The `// +build` syntax has been deprecated since Go 1.17. Let's remove it and only support `//go:build`.